unit-tests/live/extrinsics/test-imu-extrinsics.py

```diff
@@ -0,0 +1,57 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022-4 Intel Corporation. All Rights Reserved.
```

Comment:
2025

Reply:
Maybe we can rename the test name to test-imu.py?
Currently the full name is has 2 extrinsics
`found test-live-extrinsics-imu-extrinsics `

Reply:
Done